Makefile

```diff
@@ -60,10 +60,12 @@ vendor:
 	go mod tidy
 	go mod vendor
 	go mod verify
+	./hack/verify-diff.sh
```

Comment:
I don't think you want this here? This will cause `make vendor` to return an error if it updates anything. I think we only want this behaviour in the vendor test.

Reply:
Updated the script to error out only in CI.